### ISSUE TITLE
API Token Fail to Authenticate 

### ISSUE THREAD

**Issue by Jing-Bear** [grokability/snipe-it#15138]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

I'm having trouble authenticating against the API using the API token. The token expiration is set to 10 and AllowOverride is All. I'm not sure what else can be causing this issue

### Reproduction steps

1. Create API token in Snipe-IT web 
2. Use the API token to authenticate against the API

### Expected behavior

Authenticate against the endpoint and respond with 200 

### Screenshots

<img width="613" alt="Screenshot 2024-07-22 at 15 46 07" src="https://github.com/user-attachments/assets/e415025c-fe8a-42fb-93e9-3793831c3866">
![image-20240521-165002](https://github.com/user-attachments/assets/464a80c5-1895-4660-9bfd-611b08388722)
![image-20240521-172451](https://github.com/user-attachments/assets/d53d68e4-2c0e-4a3c-acd1-59a213aec3e0)

### Snipe-IT Version

5.1.5

### Operating System

Debian

### Web Server

Apache

### PHP Version

7.4.16

**Comment by snipe** [comment]
If you generate a new token and use that one, does it work? Does the user you're authenticating as have the ability to view a list of users?

**Comment by Jing-Bear** [comment]
New tokens do not work and the user can view the list of users via web at the /users endpoint.

**Comment by snipe** [comment]
I can't reproduce this anywhere, and I know we have a ton of customers with API integrations, so I can't help but feel like it has to be something on your end, either through postman or your server/network.

If you run the GET request via cURL, do you see the same thing?

```
curl --location --request GET 'http://YOUR-URL/api/v1/users' \
--header 'Content-Type: application/x-www-form-urlencoded' \
--header 'Accept: application/json' \
--header 'Authorization: Bearer YOUR-BEARER-TOKEN' \'
```

**Comment by Jing-Bear** [comment]
Yes. It is returning {"error": "Unauthorized or unauthenticated"} and I've tried with both Content-Type: application/json and application/x-www-form-urlencoded

**Comment by snipe** [comment]
Hi there - I know this is a pretty old issue, but I wanted to check and see if you ever got this resolved? If so, can you please close the ticket? If not, can you update us on what version of Snipe-IT you're currently using, so we can see if it's still an issue on latest?

:) Thanks!

**Comment by snipe** [comment]
Hi there - We haven't heard back in a bit, so I'm going to close this ticket for now, but will re-open it if you're still having issues.

![hackerman-hacking](https://user-images.githubusercontent.com/197404/31936003-27a7f6d8-b865-11e7-8f5e-eb03062925f4.gif)

**Comment by isaitgirl** [comment]
The issue still happening, using Snipe-IT default docker compose and installation.

I've created Personal Access Tokens for admin user and all API calls return the 

```
< HTTP/1.1 401 Unauthorized
< Date: Fri, 06 Feb 2026 18:37:28 GMT
< Server: Apache/2.4.58 (Ubuntu)
< Cache-Control: max-age=0, must-revalidate, no-cache, no-store, private
< X-Clockwork-Id: 01KGT3XB79QH0B3TM7EJ8RCRRT
< X-Clockwork-Version: 9
< X-Clockwork-Path: _debugbar/clockwork/
< phpdebugbar-id: 01KGT3XB79QH0B3TM7EJ8RCRRT
< Vary: Origin
< Pragma: no-cache
< Expires: Sun, 02 Jan 1990 00:00:00 GMT
< X-Content-Type-Options: nosniff
< Feature-Policy: accelerometer 'none';autoplay 'none';camera 'none';display-capture 'none';document-domain 'none';encrypted-media 'none';fullscreen 'none';geolocation 'none';sync-xhr 'none';usb 'none';xr-spatial-tracking 'none'
< Referrer-Policy: same-origin
< X-Frame-Options: DENY
< Set-Cookie: snipeit_session=qZ7FQeARMM8nHgPVLnVTamrptle0KqXmAXcrdBeq; expires=Sun, 15 Feb 2026 02:37:28 GMT; Max-Age=720000; path=/; httponly; samesite=lax
< Transfer-Encoding: chunked
< Content-Type: application/json
<
* Connection #0 to host snipeit.dinamize.local left intact
{"error":"Unauthorized or unauthenticated."}
```

I've tried to create api keys using php artisan and also web interface. Both generate keys, but none of them works for a simple:

```
curl -sv -XGET \
http://snipeit.mydomain.local:8000/api/v1/hardware \
-H 'Content-Type: application/json' \
-H 'Accept: application/json' \
-H 'Authentication: Bearer <token>'
```

Also tried with Content-Type: application/x-www-form-urlencoded


I don't know what else to try.
